testfixtures/logcapture.py

```diff
@@ -106,17 +107,17 @@ def uninstall_all(cls):
         "This will uninstall all existing :class:`LogHandler` objects."
         for i in tuple(cls.instances):
             i.uninstall()
-        
+
```

Comment:
My original email specifically mentioned this error (with two others): 
"W293 blank line contains whitespace" (which is the present thing you're commenting on) and you said "would be great to have a pull request for these". 
Admittedly, it was all over the place and easy to change. If ever this was accepted, it would be trivial to keep it clean, it's not like it's extra work :)